bundle/manifests/hw-event-proxy-operator.clusterserviceversion.yaml

```diff
@@ -19,7 +19,7 @@ metadata:
     capabilities: Basic Install
     categories: Event
     certified: "false"
-    containerImage: quay.io/redhat-cne/hw-event-proxy:4.10
+    containerImage: quay.io/openshift/origin-baremetal-hardware-event-proxy:4.10
     createdAt: "2021-12-20"
     description: This software manages the lifecycle of the hw-event-proxy container.
     olm.skipRange: '>=4.3.0-0 <4.10.0'
```

Comment:
This should '>=4.3.0-0 <4.10.0'
and target  release-4.11 branch

Reply:
updated to `'>=4.3.0-0 <4.11.0'`